scripts/run-bats-tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+# This script updates bats git submodules, runs two vault instances,
+# then runs bats tests and after all kills vault instances.
+
+git submodule update --init --recursive
+
+FILTER=$1
+
+vault server -dev -dev-root-token-id="root" > /dev/null 2>&1 &
```

Comment:
We should start this instance on a non-default port too.

Otherwise, if we use the default port and the developer already has an instance running on that port, the tests will delete all of the developer's entries.

Reply:
Done